index.md

```diff
@@ -6,17 +6,17 @@ weight: 0
 ![Web interface screenshot]({{ site.github.url }}/public/img/web-screenshot.png)
 
 Zipkin is a distributed tracing system. It helps gather timing data needed to
-troubleshoot latency problems in microservice architectures. It manages both the
+troubleshoot latency problems in micro-service architectures. It manages both the
```

Comment:
microservice is more commonly used than micro-service https://en.wikipedia.org/wiki/Microservices